setup.cfg

```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 3.3.4
+current_version = 3.3.5
```

Comment:
>.5 [](http://example.com/codeflow?start=21&length=2)

Please do not upgrade the iotedgedev version just yet. We will publish all the Hackathon changes at once at the end of the week and will bump the version then.